src/components/structures/InteractiveAuth.js

```diff
@@ -184,10 +185,6 @@ export default createReactClass({
                 errorText: null,
                 stageErrorText: null,
             });
-        } else {
```

Comment:
comment here feels like it would be useful as otherwise I'll probably come back to this in 6 months time and wonder why we don't set the busy flag to false.

Reply:
Good idea, comment added